doc/changelogs.md

```diff
@@ -151,3 +151,16 @@ will be most convenient for our users.
 [gfm]: https://guides.github.com/features/mastering-markdown/
 [keep_a_changelog]: https://keepachangelog.com/en/1.0.0/
 [gitlab_changelog]: https://docs.gitlab.com/ee/development/changelog.html
+
+## Alternatives
+
```

Comment:
I see no reason to use such an approach. Especially after cooking few final logs. If we use entries multiple times and it contains typos (which is frequently the case), then we need to fix the final draft multiple times.

Please, provide some reasoning here.

Reply:
I don't propose to replace our current process (I think it is the best for us, no reason to change). I just want to mention all possible alternatives.

Reply:
I think this is not an alternative due to the reasons I've mentioned above.

Reply:
How should I call the section to don't make it confusing? Alternatives considered?

Reply:
I think we should not mention wrong ways.

Reply:
Okay, thanks for the opinion. I still think that we should not only describe 'how' we doing things, but also 'why'. That's the common principle in development. The example from our [commit message suggestions](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message):

> 8. Use the body to explain **what and why** vs. how.

Reply:
I tried to make it as explicit as possible that this alternative is not a proposal to use it. See the list of changes below.

Reply:
I think that with given precautions it should look good. I'll resolve this thread.